README.md

```
@@ -1,36 +1,80 @@
-  - [License](#license)
-  - [Legal Notice](#legal-notice)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started
 
-## Installation
+### Prerequisites
 
-You can install the SMS Gateway CLI in two ways:
+* Go 1.23+ (for building from source)
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)